docs/permissions_list.md

```diff
@@ -203,7 +203,7 @@
         </tr>
         <tr>
             <td> changeHolderPercentage() </td>
-            <td> onlyOwner() </td>
+            <td> withPerm(ADMIN) </td>
```

Comment:
This can be clubbed with `setAllowPrimaryIssuance()`'s permission using `rowspan`

Reply:
`<tr>
            <td> setAllowPrimaryIssuance() </td>
            <td rowspan=2> withPerm(ADMIN) </td>
        </tr>
        <tr>
            <td> changeHolderPercentage() </td>
        </tr>`
Like this?

Reply:
yes, that should work.